packages/mdc-top-app-bar/_mixins.scss

```diff
@@ -51,25 +51,23 @@
 
 @mixin mdc-top-app-bar-mobile-breakpoint_($mobile-breakpoint: $mdc-top-app-bar-mobile-breakpoint) {
   @media (max-width: $mobile-breakpoint) {
-    .mdc-top-app-bar {
-      .mdc-top-app-bar__row {
-        min-height: $mdc-top-app-bar-mobile-row-height;
-      }
+    .mdc-top-app-bar__row {
+      height: $mdc-top-app-bar-mobile-row-height;
+    }
 
-      .mdc-top-app-bar__section {
-        padding: $mdc-top-app-bar-mobile-section-padding;
-      }
+    .mdc-top-app-bar__section {
+      padding: $mdc-top-app-bar-mobile-section-padding;
+    }
 
-      .mdc-top-app-bar__title {
-        @include mdc-rtl-reflexive-box(padding, left, $mdc-top-app-bar-mobile-title-left-padding);
-      }
+    .mdc-top-app-bar__title {
+      @include mdc-rtl-reflexive-box(padding, left, $mdc-top-app-bar-mobile-title-left-padding);
     }
 
     .mdc-top-app-bar--short {
       transition: width 200ms $mdc-animation-standard-curve-timing-function;
 
       .mdc-top-app-bar__row {
-        min-height: $mdc-top-app-bar-mobile-row-height;
+        height: $mdc-top-app-bar-mobile-row-height;
```

Comment:
I suspect this rule is also not needed? Both the non-mobile short row rule and the default mobile breakpoint rule already use this value for height.